oxidation/libparsec/crates/protocol/schema/authenticated_cmds/pki_enrollment_accept.json

```diff
@@ -0,0 +1,52 @@
+[
+    {
+        "label": "PkiEnrollmentAccept",
+        "major_versions": [
+            2,
+            3
+        ],
+        "req": {
+            "cmd": "pki_enrollment_accept",
+            "other_fields": [
+                {
+                    "name": "accept_payload",
```

Comment:
Should keep the comment from the python schema:
```
// Signature should be checked before loading
```

Reply:
done